.gitignore

```diff
@@ -1,2 +1,3 @@
-vendor
-composer.lock
+/vendor/
+/composer.lock
```

Comment:
I'm agree too that it should be considered as a library. @ogizanagi what would be the benefits of having the lock file in the repo ?